src/sdkg.rs

```diff
@@ -178,11 +134,11 @@ impl From<bincode::Error> for Error {
     }
 }
 
-impl Error {
-    fn encrypt<E: ToString>(err: E) -> Error {
-        Error::Encrypt(err.to_string())
-    }
-}
+// impl Error {
```

Comment:
Why did you comment this out?

Reply:
The old threshold crypto lib's `encrypt` with rng returned a `Result` enum while our `encrypt_with_rng()` is a non-result type. Thus the variant ` Error::Encrypt` was unused.